src/test/java/com/google/api/generator/spring/composer/SpringAutoConfigClassComposerTest.java

```diff
@@ -33,10 +34,21 @@ public void setUp() {
   }
 
   @Test
-  public void generateAutoConfigClazzTest() {
+  public void generateAutoConfigClazzGrpcTest() {
     GapicClass clazz =
         SpringAutoConfigClassComposer.instance().generate(this.context, this.echoProtoService);
-    String fileName = clazz.classDefinition().classIdentifier() + ".golden";
+    String fileName = clazz.classDefinition().classIdentifier() + "Grcp.golden";
```

Comment:
Small fileName typo here, and in the generated file
```suggestion
    String fileName = clazz.classDefinition().classIdentifier() + "Grpc.golden";
```

Reply:
Ah, good catch. thanks!